tests/managed-service/test_post_installation_state.py

```diff
@@ -0,0 +1,129 @@
+import logging
```

Comment:
don't we want the checks to be part of the deployment test itself?

Reply:
Moved the important checks to ocs_install_verification